src/shared/components/Resources/ResourceList.tsx

```diff
@@ -24,24 +30,51 @@ const ResourceList: React.FunctionComponent<ResourceListProps> = ({
   const { from, size } = paginationSettings;
   const totalPages = Math.floor(total / size);
   const current = Math.floor((totalPages / total) * from + 1);
+  const [selectedResource, setSelectedResource] = React.useState(
+    null as Resource | null
+  );
+  console.log({ selectedResource });
   return (
-    <List
-      className="resources-list"
-      loading={loading}
-      header={
-        <p className="result">{`Found ${total} resource${total > 1 && 's'}`}</p>
-      }
-      dataSource={results}
-      pagination={{
-        total,
-        current,
-        onChange: paginationChange,
-        pageSize: DEFAULT_PAGE_SIZE,
-      }}
-      renderItem={(resource: ResourceItemProps) => (
-        <ResourceItem key={resource.id} {...resource} />
+    <React.Fragment>
+      <List
+        className="resources-list"
+        loading={loading}
+        header={
+          <p className="result">{`Found ${total} resource${total > 1 &&
+            's'}`}</p>
+        }
+        dataSource={results}
+        pagination={{
+          total,
+          current,
+          onChange: paginationChange,
+          pageSize: DEFAULT_PAGE_SIZE,
+        }}
+        renderItem={(resource: Resource) => (
+          <ResourceItem
+            key={resource.id}
+            name={resource.name}
+            {...resource}
+            onClick={() => setSelectedResource(resource)}
+          />
+        )}
+      />
+      {typeof window !== 'undefined' && (
+        <Drawer
+          width={640}
+          placement="right"
+          onClose={() => setSelectedResource(null)}
+          visible={!!selectedResource}
+        >
+          {!!selectedResource && (
+            <div>
+              <h2>{selectedResource.name}</h2>
+              <ReactJson src={selectedResource.raw} />
```

Comment:
what happens on the server when `ReactJson` has no value? does it simply not render?